packages/template-tab-navigation-ng/src/app/app.component.html

```diff
@@ -1,18 +1,41 @@
-<TabView androidTabsPosition="bottom">
+<BottomNavigation>
+    <TabStrip>
+        <!--
+            Besides the approach demonstrated below there is alternative shorthand syntax for setting TabStripItem title and icon:
 
-    <page-router-outlet
-        *tabItem="{title: 'Home', iconSource: getIconSource('home')}"
-        name="homeTab">
-    </page-router-outlet>
+            <TabStripItem title="Home" iconSource="font://&#xf015;" />
```

Comment:
I don't think we should promote this syntax when you use font icons. It's highly unusual to want the title to be with a font icon font.

Reply:
I agree, actually thought the same yesterday while explaining pros&cons of each syntax to the team -- forgot to update the comment with app resource value :)